prow/plugins/golint/golint.go

```diff
@@ -70,17 +70,8 @@ type githubClient interface {
 	ListPullRequestComments(org, repo string, number int) ([]github.ReviewComment, error)
 }
 
-const defaultConfidence = 0.8
-
-func minConfidence(g *plugins.Golint) float64 {
-	if g == nil || g.MinimumConfidence == nil {
-		return defaultConfidence
-	}
-	return *g.MinimumConfidence
-}
-
 func handleGenericComment(pc plugins.PluginClient, e github.GenericCommentEvent) error {
-	return handle(minConfidence(pc.PluginConfig.Golint), pc.GitHubClient, pc.GitClient, pc.Logger, &e)
+	return handle(*pc.PluginConfig.Golint.MinimumConfidence, pc.GitHubClient, pc.GitClient, pc.Logger, &e)
```

Comment:
This design has already caused an hook-wide outage, which is the key difference with other parts of the config.

Reply:
Is that really different than how we access any of the other plugins whose structs we hold as pointers? (Also note the issue to move them all to pointers)

Reply:
I'll have the same recommendation as soon as those other plugins cause an outage.